relay-profiling/src/lib.rs

```diff
@@ -0,0 +1,34 @@
+use serde::Deserialize;
```

Comment:
Should we add some crate-level documentation here? See for example https://github.com/getsentry/relay/blob/d88d327c9f919c4a7c2d9f32cc365bd56c3d3e5b/relay-metrics/src/lib.rs#L1-L93.